.github/workflows/test-integration.yml

```diff
@@ -15,29 +15,32 @@ jobs:
       fail-fast: false
       matrix:
         entry:
-          - { opensearch_version: "", java: 11 }
+          - { opensearch_version: 1.0.0, java: 11 }
+          - { opensearch_version: 1.0.1, java: 11 }
+          - { opensearch_version: 1.1.0, java: 11 }
+          - { opensearch_version: 1.2.0, java: 11 }
+          - { opensearch_version: 1.2.1, java: 11 }
+          - { opensearch_version: 1.2.2, java: 11 }
+          - { opensearch_version: 1.2.3, java: 11 }
+          - { opensearch_version: 1.2.4, java: 11 }
+          - { opensearch_version: 1.3.0, java: 11 }
```

Comment:
Sorry I missed this earlier, you can also add `1.3.1` in here - on `1.x` branch we have https://github.com/opensearch-project/opensearch-java/blob/1.x/.github/workflows/test-integration.yml.

Reply:
https://github.com/opensearch-project/opensearch-java/pull/154/commits/f6af806afdf2a7093a49a01057ef54bb3ae5776a